index.js

```diff
@@ -1,18 +1,131 @@
+/* This is the main program for Project 1 - Guess the number - 
+  This program will display the number a user is thinking of
+  
+  Code author: Denis Poirier, BCA Sum2019, drpoirier 
+    Date Started:     Thursday Feb 6 2019
+    Date Completed:   in-process
+    Last Date Revised:Sunday Feb 9 2019    
+*/
+
 const readline = require('readline');
 const rl = readline.createInterface(process.stdin, process.stdout);
 
+// hard coding to min & max to begin with ; later these will be changed as user inputs for what they want
+let min=1;    // not inclusive of 0
+let max=100;  // at first limit the max to 100, then increase it to 500 or give user a choice 
+let cpuGuessNum = 0; // setting to zero as a default ? (also makes it a global so it can change thruought)
+let range = ((max - min)+1); // range = ((max# - min)+1) so it's not 0 
+
+// function source = MDN - The maximum (max) is inclusive and the minimum (min) is inclusive -
+//   This function calculates the best/shortest number of tries
+let maxTries = (Math.round((Math.log2((max)+1))));
+
+function getRandomIntInclusive(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     rl.question(questionText, resolve);
   });
 }
 
 start();
-
+  
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
+  console.log("Let's play a game where you (human user) make up a number and I (computer program) try to guess it.")
+  console.log("Please think of a number between " + min + " and " + max + " (inclusive), in this " + range + " number range.");
```

Comment:
Instead of having multiple ensuing console.logs, you can always use a [newline character](https://www.w3schools.com/jsref/jsref_regexp_newline.asp) to simply make a new line.
i.e:
```js
console.log( "log 1" , '\n',  "log 2" , '\n',  "log 3" );
```
OR
```js
console.log({ a: 1 }, '\n', { b: 3 }, '\n', { c: 3 })
```
will work as well.